### PR TITLE
PR: 2022 Update/Simplification! 

### PR DESCRIPTION
**Pull request by nelsonic**
This PR: 
+ [x] Updates the tutorial to latest `Elixir`, `Phoenix` and `LiveView` versions.
+ [x] Removes the setup/config code that's no longer required.
+ [x] Re-numbers the sections
+ [x] Adds intro image of `LiveBeats` to give people a flavour of what's possible with `LiveView`.

### CODE REVIEW

**Comment by nelsonic** [comment]
_Love_ it when a PR removes more code than it ads. 😍 
Feels _rare_ these days. 💭 
But now that `LiveView` is included with `Phoenix` and there's no setup required ... 🎉

File: README.md [hunk 1]
```diff
@@ -289,111 +284,9 @@ in your web browser.
 
 ![welcome-to-phoenix](https://user-images.githubusercontent.com/194400/76152198-ae210200-60b4-11ea-956f-68935daddfe0.png)
 
-> 🏁 Snapshot of code at the end of Step 1:
```
Comment (nelsonic) [inline README.md:292]: So glad to be dropping all these setup/config steps! #52 🎉

**Comment by nelsonic** [comment]
@SimonLab hopefully this update fixes #52 that you opened last year. 🤞 
Please review/merge when you can. Thanks.

**Review by ajmeese7 — APPROVED** [review]
Great stuff, love the elegant simplicity!

**Pull request merged**